app/assets/src/components/visualizations/heatmap/heatmap.scss

```diff
@@ -99,24 +99,25 @@
       }
 
       .metadataAddLabel {
-        @include font-label-xs;
+        @include font-label-s;
         fill: $medium-grey;
-        opacity: 0;
         cursor: pointer;
         text-anchor: end;
       }
 
       &:hover {
         .metadataAddLine {
-          stroke: $medium-grey;
+          stroke: $primary-light;
         }
 
         .metadataAddIcon {
-          opacity: 0.5;
+          // The filter is defined in heatmap.js
+          filter: url("#blue");
```

Comment:
I guess using filter mean neither `fill` or `stroke` worked for this icon?

Reply:
It seems that the plus icon is an svg embedded in the heatmap, so applying `fill` or `stroke` didn't work. @MarkAZhang helped me find the filter workaround!